vendor/meilisearch/meilisearch-php/CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ The draft release description is therefore generated and corresponds to all the
 About this automation:
 - As the draft release description is generated on every push on `master`, don't change it manually until the final release publishment.
 - If you don't want a PR to appear in the release changelogs: add the label `skip-changelog`. We suggest removing PRs updating the README or the CI (except for big changes).
-- If the changes you are doing in the PR are breaking: add the label `breaking-change`. In the release tag, the minor will be increased instead of the patch. The major will never be changed until [MeiliSearch](https://github.com/meilisearch/MeiliSearch) is stable.
+- If the changes you are doing in the PR are breaking: add the label `breaking-change`. In the release tag, the minor will be increased instead of the patch. The major will never be changed until [Meilisearch](https://github.com/meilisearch/meiliSearch) is stable.
```

Comment:
```suggestion
- If the changes you are doing in the PR are breaking: add the label `breaking-change`. In the release tag, the minor will be increased instead of the patch. The major will never be changed until [Meilisearch](https://github.com/meilisearch/meilisearch) is stable.
```